docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -10,20 +10,24 @@ This page serves a reference on Web UI features and their usage.
 
 ## Joining an active session
 
-The Web UI allows you to list and join active SSH sessions via a web-based terminal.
+The Teleport Web UI allows you to list and join active SSH sessions using a web-based terminal.
 
-Any active SSH sessions that you are allowed to list will be listed on the "Active Sessions" page, which can be 
-accessed from the navigation bar on the left side. If you don't see the "Active Sessions" tab, it means that your user's role doesn't grant
-you `list` access for the `ssh_session` resource. Please refer to the [Teleport Access Controls Reference](../access-controls/reference.mdx)
-and make sure your role has all the necessary permissions.
+You can view the active SSH sessions that you are allowed to list by clicking **Active Sessions** in the navigation sidebar. 
+You can only see Active Sessions if you are assigned a role with `list` access for the `ssh_session` resource. 
```

Comment:
Should "Active Sessions" be capitalized here? Not sure why we're capitalizing this but not "moderated sessions". If we're referring to a UI view, I think we should say "the Active Sessions view".